src/Enterspeed.Source.UmbracoCms.V8/Services/IEnterspeedGuardService.cs

```diff
@@ -0,0 +1,11 @@
+using Umbraco.Core.Models;
+using Umbraco.Core.Models.PublishedContent;
+
+namespace Enterspeed.Source.UmbracoCms.V8.Services
+{
+    public interface IEnterspeedGuardService
+    {
+        bool CanPublish(IPublishedContent content, string culture);
+        bool CanPublish(IDictionaryItem dictionaryItem, string culture);
```

Comment:
Should we call it CanIngest() as we are not really publishing content to Enterspeed but rather ingesting it?

Reply:
Renamed